src/modules/fs.rs

```diff
@@ -61,3 +64,43 @@ pub async fn copy_dir(from: impl AsRef<Path> + 'static, to: impl AsRef<Path> + '
 
     Ok(())
 }
+
+pub fn get_home_dir() -> Result<PathBuf> {
+    let home_str = if cfg!(unix) {
+        let temp = std::env::var("SUDO_USER")?;
+        let mut da_str = "/home/".to_string();
+        da_str.push_str(&temp);
```

Comment:
Line 70 causes error if `$SUDO_USER` is not present. Possible fix: 

```suggestion
        let mut da_str = "/home/".to_string();
        
        if let Ok(sudo_user) = std::env::var('SUDO_USER') {
            da_str.push_str(&temp);
        }
```